pkg/controller/envoy/ads_event.go

```diff
@@ -178,7 +184,13 @@ func (svc *ServiceEvent) handleEdsResponse(rsp *service_discovery_v3.DiscoveryRe
 		if err = anypb.UnmarshalTo(resource, loadAssignment, proto.UnmarshalOptions{}); err != nil {
 			continue
 		}
-		apiStatus := svc.DynamicLoader.ClusterCache.GetApiCluster(loadAssignment.GetClusterName()).ApiStatus
+		svc.ack.ResourceNames = append(svc.ack.ResourceNames, loadAssignment.GetClusterName())
```

Comment:
shouldnot we move this line down after `if cluster == nil {`

Reply:
yeah，if cluster ==nil， it seems that don't need to subscribe anymore.
will fix it